restful_api/controllers/2

```diff
@@ -0,0 +1,32 @@
+#-*- coding: utf-8 -*-
+from odoo import http
+from odoo.http import request
+import json
+
+class RestfulApi(http.Controller):
+    @http.route('/restful_api/restful_api', auth='public')
+    def index(self, **kw):
+        vendor = request.env['res.partner'].search_count([])
+        return "Totally %s vendor avilable now!"
+
+    @http.route('/api/products', type='json', csrf=False, auth='public', methods=['GET', 'POST'])
```

Comment:
Why is csrf set to false? Isn't it better to keep it to true?

Reply:
To avoid hassle while development this csrf set as False, now i changed as True